transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/BiosampleTransformations.scala

```diff
@@ -32,10 +36,8 @@ object BiosampleTransformations {
       dateObtained = biosampleInput.tryRead[LocalDate]("date_obtained"),
       derivedFromBiosampleId =
         biosampleInput.tryRead[String]("part_of").map(CommonTransformations.transformId),
-      // TODO needs biosampleType join
-      anatomicalSite = "site",
-      // TODO needs biosampleType join
-      biosampleType = "type",
+      anatomicalSite = joinedType.read[String]("term_id"),
+      biosampleType = joinedType.read[String]("classification"),
       samplePreservationState = biosampleInput.tryRead[String]("preservation_method"),
       seeAlso = biosampleInput.tryRead[String]("url"),
       donorId = CommonTransformations.transformId(biosampleInput.read[String]("donor")),
```

Comment:
There is one more join lurking below (needs Experiment info). I'm going to knock it out while I'm in here.

Reply:
Errr, actually not. It looks like another case where we could end up with many replication-related values for a single sample, will make a ticket for it.

Reply:
Is this ready for review then?

Reply:
Yup!